homeassistant/components/ryse/strings.json

```
@@ -0,0 +1,24 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Pairing RYSE device",
+        "description": "Please press the PAIR button on the RYSE device and then click OK.",
```

the button in the config flow is named submit. And by using the word `press` we also adhere to the fact, that users can either click or tapp the button depending if it is a PC or a mobile device.
```suggestion
        "description": "Please press the PAIR button on the RYSE device and then press submit.",
```